Notify/Notify/Notify/AppShell.xaml.cs

```diff
@@ -35,6 +43,35 @@ public AppShell()
             }
         }
 
+        private void ConnetivityChangedHandeler(object sender, ConnectivityChangedEventArgs e)  // TODO: move into Notify.Android
+        {
+            var connectivityManager = (ConnectivityManager)Android.App.Application.Context.GetSystemService(Context.ConnectivityService);
+            var capabilities = connectivityManager.GetNetworkCapabilities(connectivityManager.ActiveNetwork);
+
+            if (capabilities.HasTransport(TransportType.Wifi))
+            {
+                var wifiManager = (WifiManager)Android.App.Application.Context.GetSystemService(Context.WifiService);
+                string ssid = wifiManager.ConnectionInfo.SSID;
+        
+                if (ssid == my_ssid)
+                {
+                    notificationManager.SendNotification("Wifi - Push Notification", 
+                        "You have just connected to your wifi network!");
+                    Debug.WriteLine("Wifi - Push Notification. \nSSID: " + ssid);
```

Comment:
Consider using string interpolation using $.

change WIFI word to be the same in all places, so it will be consistent.
Some places were Wifi

Reply:
I inserted string interpolation in all the necessary places.